apps/aecore/lib/aecore/structures/data_tx.ex

```diff
@@ -0,0 +1,123 @@
+defmodule  Aecore.Structures.DataTx do
+  @moduledoc """
+  Aecore structure of a transaction data.
+  """
+
+  alias Aecore.Structures.DataTx
+  alias Aecore.Keys.Worker, as: Keys
+  alias Aecore.Chain.ChainState
+  alias Aecore.Structures.SpendTx
+  alias Aeutil.Serialization
+  alias Aeutil.Parser
+
+  require Logger
+
+  @typedoc "Name of the specified transaction module"
+  @type tx_type :: SpendTx
```

Comment:
I'd choose a plural name, to indicate more possible types

Reply:
in `Aecore.Structures.Transaction` the name is different

Reply:
But then if you read some specification let say for example -> @spec init(tx_types(), ...) this would indicate to me that the first argument requires a list of types or something similar because of he plural form